spec/integration/rack_spec.rb

```diff
@@ -88,6 +88,26 @@
       specify { expect(last_response.status).to eq 200 }
       specify { expect(last_response.body).to eq "Test" }
     end
+
+    context "for an unmodified response" do
+      before do
+        get(
+          "/component/test_component",
+          {},
+          {
+            "HTTP_IF_MODIFIED_SINCE" => "Mon, 11 Apr 2016 10:39:57 GMT"
```

Comment:
@dazoakley, I was reading through the rack documentation and I see this is one of the ways described to specify a header. An alternative was using `header key, value`.

Neither of these work for me. Do you know why the above would be an issue? I'll continue to look into this.